verilog/analysis/checkers/module_filename_rule.cc

```diff
@@ -54,10 +55,20 @@ const char ModuleFilenameRule::kMessage[] =
 
 std::string ModuleFilenameRule::GetDescription(
     DescriptionType description_type) {
-  return absl::StrCat(
+  static std::string basic_desc = absl::StrCat(
```

Comment:
static const?

Reply:
Ah, that happens when one writes a quick hat at midnight :)
Fixed.